### PR TITLE
release 0.6.0

### PR DESCRIPTION
**Pull request by gxthrj**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,12 +19,32 @@
...
+- [0.6.0](#060)
 - [0.5.0](#050)
 - [0.4.0](#040)
 - [0.3.0](#030)
 - [0.2.0](#020)
 - [0.1.0](#010)
 
+# 0.6.0
+
+这个版本我们新增了一些特性，修复了一些bug,同时也对内部代码做了一些优化。
```
Comment (tao12345666333) [inline CHANGELOG.md:31]: In English?

Comment (gxthrj) [reply CHANGELOG.md]: yep

Comment (gxthrj) [reply CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,12 +19,32 @@
...
+- [0.6.0](#060)
 - [0.5.0](#050)
 - [0.4.0](#040)
 - [0.3.0](#030)
 - [0.2.0](#020)
 - [0.1.0](#010)
 
+# 0.6.0
+
+We have added some new features, fixed some bugs, and made some optimizations to the internal code.
```
Comment (tokers) [inline CHANGELOG.md:31]: Should tell people, CRDs were changed, should re-apply them.

**Comment by tokers** [comment]
@gxthrj Should also change the document to tell people the compatibility relationship between APISIX and apisix-ingress-controller.

**Review by tao12345666333 — APPROVED** [review]

**Comment by gxthrj** [comment]
> @gxthrj Should also change the document to tell people the compatibility relationship between APISIX and apisix-ingress-controller.

changed, apisix-ingress-controller 0.6 with apisix 2.6

**Review by tokers — APPROVED** [review]

**Pull request merged**